### PR TITLE
chore(infrastructure): Remove unnecessary CLI options

### PR DESCRIPTION
**Pull request by acdvorak**
Removes 3 CLI options and moves them to `constants.js`:

- `--test-dir`
- `--golden-path`
- `--gcs-bucket`

Nobody will ever need to override these values.

Having a CLI flag for a constant is confusing because it implies that users can/should override it, but it ends up breaking the tests in non-obvious ways.

### CODE REVIEW

File: test/screenshot/lib/cli-arg-parser.js [hunk 1]
```diff
@@ -19,7 +19,11 @@
 const GitRepo = require('./git-repo');
 const argparse = require('argparse');
 const fs = require('mz/fs');
-const path = require('path');
+
```
Comment (patrickrodee) [inline test/screenshot/lib/cli-arg-parser.js]: Nit: Make these all inline to follow the style of the other imports.

File: test/screenshot/lib/cli-arg-parser.js [hunk 2]
```diff
@@ -195,29 +163,25 @@ The default behavior is to always build assets before running the tests.
   }
 
   initCleanCommand_() {
+    // eslint-disable-next-line no-unused-vars
```
Comment (patrickrodee) [inline test/screenshot/lib/cli-arg-parser.js]: Why is this necessary?

File: test/screenshot/lib/cloud-storage.js [hunk 3]
```diff
@@ -16,13 +16,16 @@
...
 
-const CliArgParser = require('./cli-arg-parser');
 const Duration = require('./duration');
 const GitRepo = require('./git-repo');
 const GoogleCloudStorage = require('@google-cloud/storage');
 const LocalStorage = require('./local-storage');
 const ProcessManager = require('../lib/process-manager');
-const {ExitCode} = require('../lib/constants');
+
+const {
```
Comment (patrickrodee) [inline test/screenshot/lib/cloud-storage.js]: Nit: Inline these imports like the above comment.

File: test/screenshot/lib/cli-arg-parser.js (hunk 1 above)
Comment (acdvorak) [reply test/screenshot/lib/cli-arg-parser.js]: Done

File: test/screenshot/lib/cloud-storage.js (hunk 3 above)
Comment (acdvorak) [reply test/screenshot/lib/cloud-storage.js]: Done

File: test/screenshot/lib/cli-arg-parser.js (hunk 2 above)
Comment (acdvorak) [reply test/screenshot/lib/cli-arg-parser.js]: It's not. Removed!

**Comment by acdvorak** [comment]
I don't think the README needs to be updated, because it doesn't mention any of the CLI options I removed 😀
